### PR TITLE
feat(Row): add right text to rows

### PR DESCRIPTION
**Pull request by marcoskolodny**
Specs: [Link](https://www.figma.com/file/Be8QB9onmHunKCCAkIBAVr/%F0%9F%94%B8-Lists-Specs?type=design&node-id=0-1&mode=design&t=Swk4wuryBgzKXSeA-0)

### CODE REVIEW

**Review by pladaria — APPROVED** [review]

File: src/__screenshot_tests__/__image_snapshots__/list-screenshot-test-tsx-boxed-row-list-disabled-custom-element-with-text-1-snap.png
Comment (atabel) [inline src/__screenshot_tests__/__image_snapshots__/list-screenshot-test-tsx-boxed-row-list-disabled-custom-element-with-text-1-snap.png:1]: I think detail text should be right aligned:
image

vs spec:
image

Comment (marcoskolodny) [reply src/__screenshot_tests__/__image_snapshots__/list-screenshot-test-tsx-boxed-row-list-disabled-custom-element-with-text-1-snap.png:1]: good catch. Fixed

**Review by atabel — APPROVED** [review]

**Comment by AnaMontes11** [comment]
Captura de Pantalla 2023-06-29 a las 9 44 44

In the case of double interaction, the "Detail" text should be to the right of the divider. As part of the main switch block.

File: src/list.tsx
Comment (yceballost) [inline src/list.tsx:1]: disabled prop is not working with detail
https://tinyurl.com/2yruca8z

**Review by yceballost — DISMISSED** [review]

File: src/list.tsx
Comment (marcoskolodny) [reply src/list.tsx:1]: Good catch. Fixed :)

**Review by yceballost — APPROVED** [review]

**Pull request merged**